stingray/crossspectrum.py

```diff
@@ -638,11 +647,11 @@ class AveragedCrossspectrum(Crossspectrum):
 
     Parameters
     ----------
-    lc1: :class:`stingray.Lightcurve` object OR iterable of :class:`stingray.Lightcurve` objects
+    lc1: :class:`stingray.Lightcurve`object OR iterable of :class:`stingray.Lightcurve` objects OR :class:`stingray.EventList` object
```

Comment:
In principle, this should work for single event lists as well. It is done so that the event list is split into GTIs, each GTI is transformed into a light curve, and then processed. This is to allow Averaged*spectrum to work transparently with event lists and light curves with no further parameters.
However, I do agree this is not an ideal way to deal with this.
I'm starting to wonder if a separate function, as you suggest, would actually be better. However, this would produce a lot of code duplication.